src/partr.c

```diff
@@ -523,6 +527,7 @@ JL_DLLEXPORT jl_task_t *jl_task_get_next(jl_value_t *trypoptask, jl_value_t *q,
             assert(jl_atomic_load_relaxed(&ptls->sleep_check_state) == not_sleeping);
             uv_mutex_unlock(&ptls->sleep_lock);
             JULIA_DEBUG_SLEEPWAKE( ptls->sleep_leave = cycleclock() );
+            ptls->timing_tls.sleep_time += jl_hrtime() - tsleep0;
```

Comment:
currently `sleep_time` is a subset of `scheduler_time` (so we might want to rename `scheduler_time` to make that clear, or do the extra accounting so that we stop accumulating `scheduler_time` when we start accumulating `sleep_time`?)